packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -47,6 +49,34 @@ class DescopeWc extends BaseDescopeWc {
 
   flowState: State<FlowState>;
 
+  log = {
+    error: (message: string, description?: string) => {
+      super.logger().error(message, description);
+      this.#dispatchNext({
```

Comment:
I don't think that we should dispatch next on every log, and if so, I would communicate it differently 

lets if something happened during execution that is not a next response, it should not trigger the `onNext` callback

Reply:
maybe i should change the event name? because the intention is to know about everything that the sdk is doing specially any error. its easier to call it all "next" for simplicity. currently, all logs are tied to an action and all logs i would like to send

Reply:
there are other logs occurrences that are not tied to the next:

![image](https://github.com/descope/descope-js/assets/10514677/47f8f10f-8954-44d7-8b39-2cd49850f480)

so imo it is not easier/simpler, cuz this name affects the api we expose

I suggest we only trigger `next` when we should (aka after `next` response) or adjust the API to something more appropriate to log